### PR TITLE
CLI: follow LAMI 1.0 (add low/high properties to number objects)

### PR DESCRIPTION
**Pull request by eepp**

### CODE REVIEW

File: lttnganalyses/cli/mi.py [hunk 1]
```diff
@@ -281,7 +242,48 @@ def _eq(self, other):
...
+            obj['low'] = self._low
+
+        if self._high is not None:
+            obj['high'] = self._high
+
+        return obj
+
+    def _eq(self, other):
+        self_tuple = (self.value, self._low, self._high)
+        other_tuple = (other.value, other._low, other._high)
```
Comment (abusque) [inline lttnganalyses/cli/mi.py]: I'm not quite sure of how we want to define equality here. I feel like simple comparing `value` might also be valid, depending on the case.

File: lttnganalyses/cli/mi.py [hunk 2]
```diff
@@ -281,7 +242,48 @@ def _eq(self, other):
...
+NEG_INF = '-inf'
+POS_INF = '+inf'
+
+
+class Number(_SimpleValue):
+    CLASS = 'number'
+
+    def __init__(self, value, low=None, high=None):
+        super().__init__(value)
+        self._low = low
```
Comment (abusque) [inline lttnganalyses/cli/mi.py]: Is there any specific reason why `value` is publicly accessible, but not `_low` or  `_high`?

Comment (eepp) [inline lttnganalyses/cli/mi.py]: I didn't implement the `low` and `high` properties yet I guess.

**Comment by abusque** [comment]
Merged manually after adding low and high properties, and using properties in comparators.

**Pull request closed without merging**